specs/agents/mobile/metrics.md

```diff
@@ -0,0 +1,41 @@
+## Mobile Metrics
+
+### CPU metrics
+| Name               | Type             | Units      | Description                     |
+|--------------------|------------------|------------|---------------------------------|
+| `system.cpu.usage` | Observable Guage | percentage | A percentage value of cpu usage |
```

Comment:
```suggestion
| `system.cpu.usage` | Gauge | percentage | A percentage value of cpu usage |
```

Not sure what an "observable gauge" is vs. a "gauge". Sounds like OTel implementation details maybe?

Reply:
Are the metric names meant to align with either OTel or other Elastic APM agents?
Also, is this meant to be system-wide CPU usage, or just for the application? Should/can we be measuring both?

Reply:
AFAIK OTel also just uses `Gauge` throughout their spec, I only encountered Observable Gauge as a data type so far (e.g. [here](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.metrics.observablegauge-1?view=net-6.0)).